Python/pythonrun.c

```diff
@@ -89,6 +90,7 @@ PyRun_InteractiveLoopFlags(FILE *fp, const char *filename_str, PyCompilerFlags *
     PyObject *filename, *v;
     int ret, err;
     PyCompilerFlags local_flags;
+    static int nomem_count = 0;
```

Comment:
Please add a comment to explain *why* the variable must be static.

Do we really **have to** remember the value between two calls?

Reply:
It does not need to be static, I will correct that.